### ISSUE TITLE
Confusing install instructions

### ISSUE THREAD

**Issue by gerroon** [bobboau/Bragi-MPD#7]
Hi

Can you please add a concise install steps for those of us who are not developers.

I git clone the repo. I install websockify and the python libs via Debian repo. I enter Bragi-MPD folder
```

./run 8800 localhost:6600
zsh: no such file or directory: ./run
```

then I try
```

python -m SimpleHTTPServer                                                                                                                             ⏎
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/usr/lib/python2.7/SimpleHTTPServer.py", line 235, in <module>
    test()
  File "/usr/lib/python2.7/SimpleHTTPServer.py", line 231, in test
    BaseHTTPServer.test(HandlerClass, ServerClass)
  File "/usr/lib/python2.7/BaseHTTPServer.py", line 606, in test
    httpd = ServerClass(server_address, HandlerClass)
  File "/usr/lib/python2.7/SocketServer.py", line 417, in __init__
    self.server_bind()
  File "/usr/lib/python2.7/BaseHTTPServer.py", line 108, in server_bind
    SocketServer.TCPServer.server_bind(self)
  File "/usr/lib/python2.7/SocketServer.py", line 431, in server_bind
    self.socket.bind(self.server_address)
  File "/usr/lib/python2.7/socket.py", line 228, in meth
    return getattr(self._sock,name)(*args)
socket.error: [Errno 98] Address already in use

```

**Comment by josedpedroso** [comment]
Hi. This is what I use to run Bragi-MPD:

`websockify --daemon 8866 --web=/where/you/put/Bragi-MPD 127.0.0.1:6600`

Where "8866" is the websockified port that you have to use in your config.js (and also where Bragi-MPD is made available to your browser), and "127.0.0.1:6600" is where your MPD instance listens. "/where/you/put/Bragi-MPD" should be self-explanatory.

Then just point a browser to your.ip:8866.

**Comment by gerroon** [comment]
Thanks I will try
